db/migrate/20230710025852_add_token_to_users.rb

```diff
@@ -0,0 +1,11 @@
+class AddTokenToUsers < ActiveRecord::Migration[7.0]
+  def up
+    add_column :users, :token, :string
+    User.find_each { |user| user.save! }
```

Comment:
Modifying the user records should be done in an [after party task](https://github.com/theSteveMitchell/after_party)

Reply:
Why can't you just add a col to the user table with a rails migration instead?